actionview/lib/action_view/helpers/asset_url_helper.rb

```diff
@@ -118,16 +118,67 @@ module Helpers
     module AssetUrlHelper
       URI_REGEXP = %r{^[-a-z]+://|^(?:cid|data):|^//}i
 
-      # Computes the path to asset in public directory. If :type
-      # options is set, a file extension will be appended and scoped
-      # to the corresponding public directory.
+      # This is the entry point for all assets.
+      # When using the asset pipeline (i.e. sprockets and sprockets-rails), the
+      # behavior is "enhanced". You can bypass the asset pipeline by passing in
+      # `public_folder: true` to the options.
       #
       # All other asset *_path helpers delegate through this method.
       #
-      #   asset_path "application.js"                     # => /assets/application.js
-      #   asset_path "application", type: :javascript     # => /assets/application.js
-      #   asset_path "application", type: :stylesheet     # => /assets/application.css
-      #   asset_path "http://www.example.com/js/xmlhr.js" # => http://www.example.com/js/xmlhr.js
+      # === With the asset pipeline
+      #
+      # All options passed to asset_path will be passed to `compute_asset_path
```

Comment:
Here you could write `+asset_path+` and `+compute_asset_path+`; if you don't wrap `asset_path` around pluses, RDoc will generate a link back to this method, which is pretty useless.